frontend/src/libs/components/back-button/back-button-wrapper.tsx

```diff
@@ -0,0 +1,21 @@
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+
+import { BackButton } from './back-button.js';
+
+type BackButtonWrapperProperties = {
+  onGoBack: () => void;
+  isVisible?: boolean;
+};
+
+const BackButtonWrapper: React.FC<BackButtonWrapperProperties> = ({
+  isVisible = false,
+  onGoBack,
+}) => {
+  return (
+    <div className={getValidClassNames(!isVisible && 'visually-hidden')}>
+      <BackButton onGoBack={onGoBack} />
+    </div>
```

Comment:
Maybe it's better to implement this functionality directly in `BackButton` component and set default to be visually displayed. Seems like redundant component for now

Reply:
You can see comment from @v0ldemar01 that requested to do this